deepchem/molnet/load_function/material_datasets/load_bandgap.py

```diff
@@ -158,8 +155,7 @@ def load_bandgap(
       return my_tasks, all_dataset, transformers
 
   # First type of supported featurizers
-  supported_featurizers = ['ElementPropertyFingerprint'
-                          ]  # type: List[Featurizer]
+  supported_featurizers = ['ElementPropertyFingerprint']  # type: List[str]
```

Comment:
It would be better to use type annotations instead of type hints.  Type hints were only needed for Python 2, since it didn't support annotations.

```python
supported_featurizers: List[str] = ['ElementPropertyFingerprint']
```